src/math/minkowski_difference.rs

```diff
@@ -0,0 +1,86 @@
+/// The Minkowski distance or Minkowski metric is a metric in a normed vector space which
+/// can be considered as a generalization of both the Euclidean distance and the Manhattan distance.
+/// It is named after the Polish mathematician Hermann Minkowski.
+/// https://en.wikipedia.org/wiki/Minkowski_distance
+///
+/// This metric is useful in the field of machine learning to judge the similarity between data points.
+/// It's also employed for collision detection in game engines.
+use num_traits::Pow;
+
+pub struct Point {
+    pub x: Vec<f64>,
+}
```

Comment:
I am not sure if this is really needed. I would be fine with using _naked_ `Vec<f64>`.